src/test/scala/com/wavesplatform/state/RollbackSpec.scala

```diff
@@ -92,8 +92,8 @@ class RollbackSpec extends FreeSpec with Matchers with WithState with Transactio
               ))
           }
 
-          val stransactions1 = d.addressTransactions(sender).sortBy(_._2.timestamp)
-          val rtransactions1 = d.addressTransactions(recipient).sortBy(_._2.timestamp)
+          val stransactions1 = d.addressTransactions(sender).getOrElse(Seq.empty).sortBy(_._2.timestamp)
```

Comment:
not sure getOrElse is good here, what if both asserted version and expected result fail? the test would pass.
maybe use `.explicitGet()` here?

Reply:
+